polaris-react/src/components/Checkbox/tests/Checkbox.test.tsx

```diff
@@ -251,31 +251,6 @@ describe('<Checkbox />', () => {
       });
     });
   });
-
-  describe('Hovering the label', () => {
-    it('adds the hover class to the Backdrop onMouseOver the label', () => {
-      const checkBox = mountWithApp(<Checkbox label="checkbox" />);
-
-      const label = checkBox.find('label');
-      label!.trigger('onMouseOver');
```

Comment:
This doesn't exist anymore.

... Also, why are our tests asserting against internal implementation details?